Maria ignacio/add day night cycles

mariaignacio opened the pull request:
Closes #4

1strangequark commented on griffithScene.js:
```diff
@@ -265,7 +350,7 @@ class Gouraud_Shader extends Shader {
                 float diffuse  =      max( dot( N, L ), 0.0 );
                 float specular = pow( max( dot( N, H ), 0.0 ), smoothness );
                 float attenuation = 1.0 / (1.0 + light_attenuation_factors[i] * distance_to_light * distance_to_light );
-                
```
This is not a big deal at all, and don't feel the need to change this, but it's usually good to not commit newlines or other non-useful changes. That way the diff file will be a little cleaner.

1strangequark commented:
This looks really cool. I am noticing that the day / night cycle seems to be a little out of sync with the movements of the sun. Can this be changed such that the sky always gets brighter when the sun rises?

Also, I think it may be good to not have the sun and the moon as a single object. I was getting a little confused between the two, so perhaps keep it as just a sun for now, maybe? I think if we add a moon object later, we should make it a distinctly gray color and put it on a different orbit

1strangequark commented:
If you do want to leave the sun and the moon as the same object, I do think the color of the moon should be darkened as to make it very distinct from the bright yellow sun, but that's just my view.

mariaignacio commented:
I added a commit to remove the moon object. I also fixed the day/night cycles, however, there is a bug where if you speed up or slow down the cycle times during the night cycle the sky changes not as expected.

1strangequark reviewed (approved):
This looks great, Maria! Feel free to merge this and delete the branch. Then you can create a new branch that implements a new feature or fixes the small bug you mentioned.
